src/main.cpp

```diff
@@ -60,6 +61,7 @@ int runMixxx(MixxxApplication* pApp, const CmdlineArgs& args) {
                 &MixxxMainWindow::initializationProgressUpdate);
         pCoreServices->initialize(pApp);
         mainWindow.initialize();
+        pCoreServices->getControllerManager()->setUpDevices();
```

Comment:
this line could alternatively go in mixxxmainwindow

Reply:
At least there is an asymetie for the QMl and widget case. Can we move both to main, or do we have a good reason for the current places?